pkg/config/config.go

```diff
@@ -110,40 +154,68 @@ func NewLocalConfig() LocalConfig {
 	}
 }
 
+// newProxyLocalConfig creates an empty proxyLocalConfig struct with typeMeta populated
+func newProxyLocalConfig() proxyLocalConfig {
+	lc := NewLocalConfig()
```

Comment:
can you add a way to pass a config file path(that's not the current directory basically context use case) which will be used here?

Reply:
there is alot of work related to this in your PR, maybe you can rebase and do that, If I do it then alot of the work would have to be moved here.

Reply:
also seems out of scope

Reply:
ok

Reply:
resolving